Source/WebCore/page/Page.cpp

```diff
@@ -5426,6 +5426,21 @@ void Page::flushDeferredResizeEvents()
     });
 }
 
+void Page::startDeferringSetNeedsVisualViewportResizeEvent()
+{
+    m_shouldDeferSetNeedsVisualViewportResizeEvent = true;
+}
+
+void Page::flushDeferredSetNeedsVisualViewportResizeEvent()
+{
+    if (!m_shouldDeferSetNeedsVisualViewportResizeEvent)
+        return;
+    m_shouldDeferSetNeedsVisualViewportResizeEvent = false;
+    forEachDocument([&] (Document& document) {
```

Comment:
Style guide nit: No space between the capture list and the argument list.

https://webkit.org/code-style-guidelines/#spacing-lambda-paren

```suggestion
    forEachDocument([&](Document& document) {
```

Reply:
Also, we don't need to capture anything here.
```suggestion
    forEachDocument([](Document& document) {
```